Sources/Core/ARM/SoftwareBreakpointManager.cpp

```diff
@@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
   }
 #endif
 
+  opcode.clear();
```

Comment:
Sounds good.

On Friday, August 12, 2016, Francis Ricci <notifications@github.com> wrote:

> In Sources/Core/ARM/SoftwareBreakpointManager.cpp
> <https://github.com/facebook/ds2/pull/422#discussion_r74683024>:
>
> > @@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
> >    }
> >  #endif
> >
> > +  opcode.clear();
>
> Nope, was just not happy with push_back() on something that might already
> have a value. I'll keep it in this commit then.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/422/files/50a5cbdc6a795f7383c093429619b7c05ce45b12..7cdfc54368119ccdfb434303997ba7f05611c54e#r74683024>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADHbA52D3LHyV8TS96K04ISbbJTwU1uks5qfWgggaJpZM4JjTCI>
> .
>


-- 
Stephane Sezer